src/components/ChecImageManager.vue

```diff
@@ -24,11 +24,15 @@
           @remove="() => removeFile(file)"
         />
       </Draggable>
-      <ChecButton @click="openDialog">
-        <ChecIcon class="chec-image-manager__icon" icon="image" size="base" /> {{ $t('imageManager.chooseImages') }}
-      </ChecButton>
-      <div v-if="footnote" class="chec-image-manager__helper">
-        {{ footnote }}
+      <div v-if="maxFiles === null || maxFiles > allFiles.length">
```

Comment:
```suggestion
      <template v-if="maxFiles === null || maxFiles > allFiles.length">
```

BYO closing tag. Makes more sense not to add the block if it isn't needed?

Reply:
Good call, I've switched it out.